zephyr-env.sh

```diff
@@ -12,7 +12,7 @@
 #
 # Note: The version of zsh need to be 5.0.6 or above. Any versions below
 # 5.0.6 maybe encoutner errors when sourcing this script.
-if [ -n "$ZSH_VERSION" ]; then
+if [ -n "${ZSH_VERSION-}" ]; then
```

Comment:
Shouldn't this be `${ZSH_VERSION:-}`?

Reply:
I tested this with bash and zsh 5.1. According to Bash `man` page:
> Omitting the colon results in a test only for a parameter that is unset.

My commit message was not correct. I updated it and changed the code to check for null as well (safer variant).